src/components/ActiveVideo/Uploading/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { h, FunctionComponent } from 'preact'
+import { useEffect, useState } from 'preact/hooks'
+import { Footer } from '../Footer'
+import { Header } from '../Header'
+import { Wrapper } from '../Wrapper'
+import { BaseScreen } from '../BaseScreen'
+import { LoaderIcon } from '../assets/LoaderIcon'
+import { localised } from '~locales'
+
+interface Props {
+  videoPayload: Blob
+  nextStep: () => void
+}
+
+const Uploading: FunctionComponent<Props> = ({ nextStep }) => {
+  const [uploadProgress, setUploadProgress] = useState(0)
+
+  let interval: NodeJS.Timeout
+
+  /**
+   * This method is only intended to mimic the real upload.
+   */
+  useEffect(() => {
+    interval = setInterval(() => {
```

Comment:
This is a temporary fake upload while we work on integrating Biometrics Media Uploader.